File: CSC-291/Projects/fibonacci.py
'''
FVCproductions
October 10, 2014
Fibbonaci Sequence Algorithm
'''

def fib(n):

	if n == 0 or n == 1:
		return 1
	if n > 1:
		return fib(n-1)+fib(n-2)

element = 14
print "Fibbonaci sequence of " + str(element) + ": " +  str(fib(element))

File: CSC-510/fibonacci/Fibonacci.java
/**
 * Fibonacci Sequence
 * FVCproductions
 * 09.08.2015
 */

public class Fibonacci {
    public static long fib(int n) {
        if (n == 0) return 0;
        if (n == 1) return 1;
        else return fib(n-1) + fib(n-2);
    }
    public static void main(String[] args) {
        int N = 50; // terribly expensive but does work eventually!
        for (int i = 0; i <= N; i++) {
            System.out.println(fib(i));
        }
    }
}